models/crypto/users.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * This file contains wrapper functions to interact with the DB.
+ * This will contain the DB schema if we start consuming an ORM for managing the DB operations
+ */
+
+const logger = require('../../utils/logger')
+const firestore = require('../../utils/firestore')
+const cryptoUsersModel = firestore.collection('crypto/users/details')
+const cryptoTransaction = require('../../utils/cryptoTransaction')
+/**
+ * Fetches the data of a users in crypto site
+ * @return {Promise<usersModel|Object>}
+ */
+
+const fetchUser = async (id) => {
+  try {
+    const user = await cryptoUsersModel.doc(id).get()
+    if (user.exists) {
+      return {
+        user: {
+          ...user.data()
+        }
+      }
+    }
+    return ''
+  } catch (err) {
+    logger.error('Error retrieving user data', err)
+    throw err
+  }
+}
+
+/**
+ * Adds the data of a user to crypto site
+ * @return {Promise<usersModel|Object>}
+ */
+
+const addUser = async (req) => {
+  try {
+    const id = req.params.id
+    let userInfo = await cryptoUsersModel.where('id', '==', id).limit(1).get()
+
+    if (userInfo.empty) {
+      const userData = {
+        id,
+        firstName: req.userData.first_name,
+        lastName: req.userData.last_name,
+        coins: {
+          copper: 1000,
```

Comment:
Are these the default coins to be added?
If yes, please add it in the default config, by creating a `features.crypto` object.